moped-editor/src/views/projects/projectsListView/ProjectsListViewQueryConf.js

```diff
@@ -176,80 +167,140 @@ export const PROJECT_LIST_VIEW_QUERY_CONFIG = {
     },
     public_process_status: {
       label: "Public process status",
       searchable: false,
       type: "string",
+      defaultHidden: true,
+      showInTable: true,
     },
-    // the following column configurations only work with material table and do not have the
-    // necessary details to work with Grid Table
     task_orders: {
       type: "array",
+      sortable: true,
+      defaultHidden: true,
+      showInTable: true,
     },
     task_order_names: {
       type: "string",
+      sortable: true,
     },
+    // signal_ids
     project_feature: {
       type: "array",
+      sortable: false,
+      defaultHidden: true,
+      showInTable: true,
     },
     type_name: {
       type: "string",
+      sortable: true,
+      defaultHidden: true,
+      showInTable: true,
     },
     funding_source_name: {
       type: "string",
+      sortable: true,
+      defaultHidden: true,
+      showInTable: true,
     },
     project_note: {
       type: "string",
+      sortable: true,
+      defaultHidden: true,
+      showInTable: true,
     },
     construction_start_date: {
       type: "date",
+      sortable: true,
+      defaultHidden: true,
+      showInTable: true,
     },
     completion_end_date: {
       type: "date",
+      sortable: true,
+      defaultHidden: true,
+      showInTable: true,
     },
     project_inspector: {
       type: "string",
+      sortable: true,
+      defaultHidden: true,
+      showInTable: true,
     },
     project_designer: {
       type: "string",
+      sortable: true,
+      defaultHidden: true,
+      showInTable: true,
     },
     contractors: {
       type: "string",
+      sortable: true,
+      defaultHidden: true,
+      showInTable: true,
     },
     contract_numbers: {
       type: "string",
+      sortable: true,
+      defaultHidden: true,
+      showInTable: true,
     },
     project_tags: {
       type: "string",
+      sortable: true,
+      defaultHidden: true,
+      showInTable: true,
     },
     added_by: {
       type: "string",
+      sortable: true,
+      defaultHidden: true,
+      showInTable: true,
     },
     interim_project_id: {
       type: "Int",
+      sortable: true,
+      defaultHidden: true,
+      showInTable: true,
     },
     children_project_ids: {
       type: "array",
+      sortable: true,
+      defaultHidden: true,
+      showInTable: true,
     },
     parent_project_id: {
       type: "Int",
+      sortable: true,
+      defaultHidden: true,
+      showInTable: true,
     },
     parent_project_name: {
       type: "string",
+      sortable: true,
     },
     components: {
       type: "string",
-    }
+      sortable: true,
+      defaultHidden: true,
+      showInTable: true,
+    },
   },
-  // This object gets consumed into the GQLAbstract system, and here is the single, un-nested order_by directive. ✅
-  order_by: { updated_at: "desc" },
-  where: null,
-  or: null,
-  and: null,
-  limit: 250,
-  offset: 0,
```

Comment:
This chunk is old config that is no longer used by this view. The defaults are handled in the hooks added recently.